lib/models/init-models.js

```diff
@@ -0,0 +1,57 @@
+var DataTypes = require('sequelize').DataTypes;
+var _SequelizeMeta = require('./SequelizeMeta');
+var _booking_attempts = require('./booking_attempts');
+var _bookings = require('./bookings');
+var _cab_types = require('./cab_types');
+var _driver_location = require('./driver_location');
+var _drivers = require('./drivers');
+var _users = require('./users');
+
+function initModels(sequelize) {
+    var SequelizeMeta = _SequelizeMeta(sequelize, DataTypes);
+    var booking_attempts = _booking_attempts(sequelize, DataTypes);
+    var bookings = _bookings(sequelize, DataTypes);
+    var cab_types = _cab_types(sequelize, DataTypes);
+    var driver_location = _driver_location(sequelize, DataTypes);
+    var drivers = _drivers(sequelize, DataTypes);
+    var users = _users(sequelize, DataTypes);
```

Comment:
looks like code is getting repeated here right,
you can created a reusable utility function that does the task

e.g const getSequelizeModal = (modelName) => modelName(sequelize, DataTypes);

then declare your modals :-

const bookings = getSequelizeModal(_booking_attempts);
const drivers = getSequelizeModal(_drivers);

..../so on

also I prefer using const over var nowadays,

Reply:
Abbishek, these code get's generated automatically.

Reply:
ah, nice

you can configure it though acc to your use cases and requirements, 
we would like to use DRY approach for better clean code sustainabilities right, 

anyways not an issuefor now